features/sample.feature

```diff
@@ -15,15 +15,18 @@ Feature: Hacker News REST API validation
     Then the response status should be "200"
     And the JSON response root should be array
     When I grab "$[0]" as "id"
+    And I save the "$[0]" as "id"
     And I send a GET request to "https://hacker-news.firebaseio.com/v0/item/{id}.json" with:
       | print  |
       | pretty |
     Then the response status should be "200"
     And the JSON response root should be object
     And the JSON response should have <optionality> key "<key>" of type <value type>
+    And the JSON response should have "id" of type numeric and value "#{id}"
```

Comment:
If you agree with my previous comment then I guess we can use `{}` for placeholder, making it consistent across all steps?